analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ClearTextProtocolsAreSensitive.cs

```diff
@@ -158,6 +159,58 @@ public void Ftp()
             "FTP://anonymous@example.com" // Noncompliant
         };
     }
+
+    [XmlRoot(ElementName = "SonarProjectConfig", Namespace = "http://www.sonarsource.com/msbuild/analyzer/2021/1")]
+    public class NamespaceLikeAssignment
+    {
+        private string NamespaceLikeField = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
+        private string NamespaceLikeProperty { get; set; }
+
+        public void Foo(string NamespaceLikeArgument)
+        {
+            var NamespaceLikeVar = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
+            NamespaceLikeArgument = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
+            NamespaceLikeField = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
+            NamespaceLikeProperty = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
+        }
+
+        public void Bar(string NamespaceLikeArgument = "http://www.sonarsource.com/msbuild/analyzer/2021/123") { }
+    }
+
+    [MyAttribute("http://www.sonarsource.com/msbuild/analyzer/2021/123")]  // Noncompliant
+    [XmlRoot(ElementName = "SonarProjectConfig", Namespace = "ftp://a:a@foo.com/")]  // Noncompliant
+    public class NamespaceLikeAssignment2
+    {
+        private string NamespaceLikeField = "ftp://a:a@foo.com/";  // Noncompliant
+        private string NamespaceLikeProperty { get; set; }
+
+
+        private string NaaamespaceLikeField = "http://www.sonarsource.com/msbuild/analyzer/2021/1";  // Noncompliant
+        private string NaaamespaceLikeProperty { get; set; }
+
+        public void Foo(string NaaamespaceLikeArgument, string NamespaceLikeArgument = "ftp://a:a@foo.com/")  // Noncompliant
+        {
+            var NamespaceLikeVar = "ftp://a:a@foo.com/";  // Noncompliant
+            NamespaceLikeArgument = "ftp://a:a@foo.com/";  // Noncompliant
+            NamespaceLikeField = "ftp://a:a@foo.com/";  // Noncompliant
+            NamespaceLikeProperty = "ftp://a:a@foo.com/";  // Noncompliant
+
+            var namespaceLikeVar = "http://www.sonarsource.com/msbuild/analyzer/2021/1";  // Noncompliant
```

Comment:
why is this Noncompliant? shouldn't the check ignore casing? it's about making sure we don't raise on any sort of namespace